### PR TITLE
#161255199 Users should be able to rate an article

### PR DESCRIPTION
**Pull request by Tony-Ndichu**
### What does this PR do?
This PR adds rating functionality to Author's Haven frontend

### Description of Task to be completed?
Enable users to rate articles using 5 stars and display the average rating using similar 5 stars

### How should this be manually tested?
- After cloning the repo, cd into it
- Checkout the branch `ft-rate-an-article`
- Run `npm install`
- Run `npm start`

### What are the relevant pivotal tracker stories?
[#161255199](https://www.pivotaltracker.com/story/show/161255199)

### Screenshots (if appropriate)
Display article ratings when listed:
screenshot 2018-12-16 at 19 28 57

When rate button is clicked:
screenshot 2018-12-16 at 22 30 21

When article owner is the logged in user, the "rate" button is disabled:
screenshot 2018-12-16 at 19 29 28

When previous rating is loading:
screenshot 2018-12-16 at 22 28 23

### Checklist:
- [x] My code follows the style guidelines of this project
- [ ] My teammates have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

**Review by MartinKaburu — APPROVED** [review]
Looks good to me.

File: src/components/Login/index.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ const LoginForm = ({ ...props }) => {
         <p>
 Don&apos;t have an account?
           <Link to="signup"> Sign Up</Link>
+
```
Comment (Rodgers-M) [inline src/components/Login/index.js]: are these blank lines necessary?

Comment (Tony-Ndichu) [reply src/components/Login/index.js]: Let me remove them

File: src/containers/Rating/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+class Rating extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+    };
+  }
+
+  show = size => () => {
```
Comment (Rodgers-M) [inline src/containers/Rating/index.js]: i think this method can be given a better name to describe what it is showing i.e if its rating then `showRating` can be ideal. same with the `close` method

Comment (Tony-Ndichu) [reply src/containers/Rating/index.js]: I've renamed the functions appropriately

**Pull request merged**